domain/src/main/kotlin/no/nav/su/se/bakover/domain/beregning/Beregning.kt

```diff
@@ -72,7 +71,7 @@ data class Beregning(
         månedsberegninger.sumBy { it.beløp } <= 0
 }
 
-private fun fradragWithForventetInntekt(fradrag: List<Fradrag>, forventetInntekt: Int): List<Fradrag> {
+fun fradragWithForventetInntekt(fradrag: List<Fradrag>, forventetInntekt: Int): List<Fradrag> {
```

Comment:
Jeg hadde ønsket med et bedre navn på denne, men klarer ikke helt å komme på et.
sjekk-forventet-inntekt-opp-mot-arbeidsinntekt-og-returner-en-liste-med-fradrag-hvor-kun-den-høyeste-av-de-to-nevnte-er-med.com

Reply:
haha ja, något sånt. tror att om man ska få den så deklarativ som möjligt så får man splitta upp den lite.

Reply:
`ErstattArbeidsinntektMedForventetInntektOmStørre` ? 😅